torchaudio/__init__.py

```diff
@@ -21,6 +23,8 @@
 except ImportError:
     pass
 
+logging.getLogger().setLevel(logging.INFO)
```

Comment:
Is any of the Python code checked in use `logging`?
I might be missing, but I do not see any usage.
If it is not used, we can remove this.